### DISCUSSION TITLE
Is there way to use nat forward ip and lan connection for webrtc?

### DISCUSSION THREAD

**Discussion by krishgcek** [bluenviron/mediamtx#2397] [Questions]
Currently I have configured tcp nat for webrtc. Can I use lan ip also with it? I see [] for the ips. So I assumed it is a list.
What ip should give there ? Is it the lan  ip of the machine where mediamtx is running?

**Comment by aler9** [comment]
Hello, instructions on how to use `webrtcICEHostNAT1To1IPs` are in the README:

https://github.com/bluenviron/mediamtx#connectivity-issues

Basically you have to provide the external IP of the server. If there's a NAT between server and clients, it's the IP of the NAT.

**Reply by krishgcek** [reply]
I am using anydesk with tcp tunnel. After giving 127.0.0.1 as ip it works over it. I would also like to add my lan ip of the server along with it. Is it possible?

[a comment by an automated account is not shown]
